app/services/AuditService.scala

```diff
@@ -87,15 +72,36 @@ trait AuditService {
       detail = ContactDetailsAuditEventDetail(externalID, authProviderId, rID, originalEmail, amendedContactDetails)
     )
 
-  abstract class RegistrationAuditEventTest(auditType: String, detail: JsObject)(implicit hc: HeaderCarrier, req: Request[AnyContent])
-    extends ExtendedDataEvent(
-      auditSource = "company-registration-frontend",
-      auditType = auditType,
-      detail = detail
+
+  def emailMismatchEventDetail(externalUserId: String,
+                               authProviderId: String,
+                               journeyId: String)
+                              (implicit hc: HeaderCarrier, req: Request[AnyContent], ec: ExecutionContext): Future[AuditResult] =
+
+    sendEvent(
+      auditType = "emailMismatch",
+      detail = EmailMismatchEventDetail(externalUserId, authProviderId, journeyId)
+    )
+
+
+  def emailVerifiedEventDetail(externalUserId: String, authProviderId: String,
+                               journeyId: String,
+                               emailAddress: String, isVerifiedEmailAddress: Boolean = true,
+                               previouslyVerified: Boolean = true)
+                              (implicit hc: HeaderCarrier, req: Request[AnyContent], ec: ExecutionContext): Future[AuditResult] =
+
+    sendEvent(
+      auditType = "emailVerified",
+      detail = EmailVerifiedEventDetail(externalUserId, authProviderId, journeyId, emailAddress, isVerifiedEmailAddress, previouslyVerified)
     )
 
-  object RegistrationAuditEventTest{}
+  def questionnaireAuditEvent(questionnaireModel: QuestionnaireModel)
+                             (implicit hc: HeaderCarrier, req: Request[AnyContent], ec: ExecutionContext): Future[AuditResult] =
 
+    sendEvent(
+      auditType = "Questionnaire",
+      detail = QuestionnaireAuditEvent(questionnaireModel)
```

Comment:
Change this so that the `questionaireModel` is passed directly - the wrapper Class is no longer needed :)
```suggestion
      detail =questionnaireModel
```